src/components/check-your-email/check-your-email-controller.ts

```diff
@@ -64,6 +68,18 @@ export const checkYourEmailPost = (
       );
       return res.redirect(path);
     }
+    if (supportCheckEmailFraud()) {
+      const { sessionId, clientSessionId, persistentSessionId } = res.locals;
+      const checkEmailFraudResponse =
+        await checkEmailFraudService.checkEmailFraudBlock(
+          req.session.user.email,
+          sessionId,
+          clientSessionId,
+          persistentSessionId,
+          req
+        );
+      logger.info(`checkEmailFraudResponse: ${checkEmailFraudResponse.data}`);
```

Comment:
Just spotted a PII issue with the original too. This would dump an email address into the logs. Can this just dump the returned status please. Thanks!